.github/workflows/spelling.yml

```diff
@@ -30,15 +67,26 @@ jobs:
     steps:
     - name: check-spelling
       id: spelling
-      uses: check-spelling/check-spelling@v0.0.20
+      uses: check-spelling/check-spelling@v0.0.21
       with:
         suppress_push_for_open_pull_request: 1
         checkout: true
+        check_file_names: 1
+        spell_check_this: check-spelling/spell-check-this@prerelease
         post_comment: 0
+        use_magic_file: 1
+        extra_dictionary_limit: 10
         extra_dictionaries:
           cspell:filetypes/filetypes.txt
-          cspell:python/python.txt
+          cspell:python/src/python/python-lib.txt
+          cspell:python/src/python/python.txt
+          cspell:python/src/common/extra.txt
```

Comment:
As noted, this upgrade refreshed the `cspell` dictionaries, and `python.txt` was restructured into a number of smaller more clearly defined dictionaries.

To update this, I cleared `extra_dictionaries:`, removed `check_extra_dictionaries: ''` and then iterated until the suggestions stopped being worthwhile -- see https://github.com/check-spelling/fprime/actions/runs/3589187603 for roughly where I stopped.